### PR TITLE
Create Title Card for August 2024 USWDS event page

### PR DESCRIPTION
**Pull request by brunerae**
Automatically generated by Decap CMS

### CODE REVIEW

File: data/images/2024-uswds-monthly-call-aug-title-card.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  # https://s3.amazonaws.com/digitalgov/2024-uswds-monthly-call-aug-title-card.png
+  # Image shortcode: {{< img src=2024-uswds-monthly-call-aug-title-card >}}'
+  date     :  2024-08-12 15:37:52 -0400
+  uid      :  2024-uswds-monthly-call-aug-title-card
+  width    :  1200
+  height   :  628
+  format   :  png
+  credit   :  
+  caption  :  
+  alt      :
```
Comment (mejiaj) [inline data/images/2024-uswds-monthly-call-aug-title-card.yml:11]: **question (non-blocking):** Do we need alt text here?

The card for June included one:
https://github.com/GSA/digitalgov.gov/blob/018c4b5e0aa8977a7ebcd14a4b0d0255d5a7d1d9/data/images/2024-uswds-monthly-call-june-title-card.yml#L11

**Review by mejiaj — COMMENTED** [review]
LGTM, minor question on alt text. Good to merge/Fed Final review if not necessary.

**Review by ToniBonittoGSA — DISMISSED** [review]

**Review by mejiaj — APPROVED** [review]
LGTM. @brunerae wrapped the alt text in quotes to be consistent with the previous call.

I'm able to see the image at the top of the [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cms/image-upload/content-uploads-_inbox-2024-uswds-monthly-call-aug-title-card-png/images/).

image

File: data/images/2024-uswds-monthly-call-aug-title-card.yml (hunk 1 above)
Comment (mejiaj) [reply data/images/2024-uswds-monthly-call-aug-title-card.yml]: Added in [c8b1157](https://github.com/GSA/digitalgov.gov/pull/7856/commits/c8b11579be8f2cd5d3be423ca715b041cba916c4).

**Pull request merged**